Changefeed/ExampleEventCreator/ExampleEventCreator.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Azure.Storage.Blobs;
+using System;
+using System.IO;
+using System.Text;
+using System.Configuration;
+
+namespace ExampleEventCreator
+{
+    class ExampleEventCreator
+    {
+        //Creates container and blobs within a storage account to populate changefeed and add example events
+        static void Main(string[] args)
+        {
+            string connectionString = ConfigurationManager.AppSettings["connectionString"]; ;
+            string containerName = "test-changefeed-container";
+            BlobServiceClient blobServiceClient = new BlobServiceClient(connectionString);
+            BlobContainerClient blobContainerClient = blobServiceClient.CreateBlobContainer(containerName);               
+            
+            for(int i = 0; i < 5; i++)
+            {
+                var upStream = new MemoryStream(Encoding.ASCII.GetBytes("blob"));
+                BlobClient blobClient = new BlobClient(connectionString, containerName, "exampleblob" + i.ToString());
+                blobClient.Upload(upStream,true);
+            }
+            Console.WriteLine("Done");
+        }
```

Comment:
Let's make sure and confirm this example event creator matches the other languages too

Reply:
I believe all of our programs just create a container and upload/delete/etc blobs to it in one way or another, but Cecilia's is doing it on a local timer and I think James and I need to decide if we need to do that as well. I believe Kamil suggested we should at one point.